### ISSUE TITLE
try adding the loading gif 

### ISSUE THREAD

**Issue by resonantdoghouse** [Safenorsound/Instanews-Project-2#6]
There are two comments for trying to add and remove the loading gif.

I'd like you to try adding these using methods like .append and .detach

For example

```$('#target-to-append-to').append('<img id="loading-gif" src="path_to_img"/>');```

and

```$('#loading-gif').detach();```

https://github.com/Safenorsound/Instanews-Project-2/blob/295e4440cfba13cfbb12abc53faac0e136c0586e/js/index.js#L5
